var/spack/repos/builtin/packages/py-pyscf/package.py

```diff
@@ -11,23 +11,28 @@ class PyPyscf(PythonPackage):
     """PySCF is a collection of electronic structure programs powered
     by Python."""
 
-    homepage = "https://sunqm.github.io/pyscf/"
+    homepage = "https://pyscf.org"
     git = "https://github.com/pyscf/pyscf"
 
     maintainers("naromero77")
 
+    version("2.1.1", tag="v2.1.1")
+    version("2.1.0", tag="v2.1.0")
     version("2.0.1", tag="v2.0.1")
     version("1.7.5", tag="v1.7.5")
     version("1.7.3", tag="v1.7.3")
 
     # dependencies
+    depends_on("cmake@3.10:", type="build", when="@2.1:")
     depends_on("cmake@2.8:", type="build")
+    depends_on("python@3.6:", type=("build", "run"), when="@2.1:")
     depends_on("python@2.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-numpy@1.8.0:", type=("build", "run"))
     depends_on("py-numpy@1.13.0:", type=("build", "run"), when="@2:")
     conflicts("py-numpy@1.16:1.17", when="@2:")
     depends_on("py-scipy@0.12:", type=("build", "run"))
+    depends_on("py-scipy@0.19:", type=("build", "run"), when="@2.1:")
```

Comment:
Where do you see these scipy version requirements?

Reply:
Should also be a requirement like:
```python
depends_on("py-scipy@:1.1.0", when="^python@:3.7", type=("build", "run"))
```

Reply:
This package follows the"installation from source" instructions from the official documentation at https://pyscf.org/install.html#compiling-from-source-code . This is the best location I could find for up to date version requirements.
```
Prerequisites for manual install are

    CMake >= 3.10

    Python >= 3.6

    Numpy >= 1.13

    Scipy >= 0.19

    h5py >= 2.7
```

I do not understand your request to add a python version dependency to py-scipy usage. Where is this indicated in the pyscf docs? Is it flagged by spack CI?

Reply:
Documentation is meaningless to me and usually doesn't get updated very often. I check pyproject.toml/setup.py/setup.cfg which is what pip checks. Look at the `_scipy_version` stuff in `setup.py` to see what I mean.

That's not to say that the documentation isn't useful and we shouldn't keep the lower bound, just that if there's a bug in the docs, no one notices or reports an issue because most people pip install it.

Reply:
Updated with requested requirement. This is darwin specific in setup.py so I added that dependency